src/projekt_zaliczeniowy.Web/Program.cs

```diff
@@ -10,13 +10,15 @@
 using FastEndpoints.ApiExplorer;
 using Microsoft.OpenApi.Models;
 using Serilog;
+using projekt_zaliczeniowy.Infrastructure.Repositories;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Host.UseServiceProviderFactory(new AutofacServiceProviderFactory());
 
 builder.Host.UseSerilog((_, config) => config.ReadFrom.Configuration(builder.Configuration));
 
+builder.Services.AddScoped<IRestaurantRepository, RestaurantsRepository>();
```

Comment:
To powinno się znaleźć w DefaultInfrastructureModule

Prawdopodobnie coś takiego:
`builder
      .RegisterType<RestaurantsRepository>()
      .As<IRestaurantRepository>()
      .InstancePerLifetimeScope();`